docs/source/details/backendconfig.rst

```diff
@@ -119,6 +122,12 @@ Explanation of the single keys:
   Please refer to the `official ADIOS2 documentation <https://adios2.readthedocs.io/en/latest/engines/engines.html>`_ for the available engine parameters.
   The openPMD-api does not interpret these values and instead simply forwards them to ADIOS2.
 * ``adios2.engine.usesteps``: Described more closely in the documentation for the :ref:`ADIOS2 backend<backends-adios2>` (usesteps).
+* ``adios2.engine.preferred_flush_target`` Only relevant for BP5 engine, possible values are ``"disk"`` and ``"buffer"`` (default: ``"disk"``).
```

Comment:
I've called this `preferred_flush_target` since it is only implemented by BP5. Otherwise, we will see user requests `"I specified flush_target=disk, why is BP4 buffering the data anyway?"`.
I don't really want to explicitly use the BP5 name, e.g. `bp5_flush_target=disk` because other engines might in future implement this too.